polling_stations/apps/api/postcode.py

```diff
@@ -57,6 +60,8 @@ def generate_advance_voting_station(self, routing_helper):
     def get_ee_wrapper(self, postcode, rh, query_params):
         if rh.route_type == "multiple_addresses":
             return EmptyEveryElectionWrapper()
+        if rh.elections_response:
+            return StaticElectionsAPIElectionWrapper(rh.elections_response)
         kwargs = {}
         query_params = parse_qs_to_python(query_params)
         if include_current := query_params.get("include_current", False):
```

Comment:
I don't think there's anything we can do about this, but as long as there are parquet files I think these lines won't be hit. Do we rely on the `include_current` flag somewhere else when we're calling the WDIV API?

Reply:
Decided to park until later